### ISSUE TITLE
Define "Multi-component reacting NS" in JOSS paper

### ISSUE THREAD

**Issue by WilkAndy** [uw-comphys/opencmp#9]
This could mean different things to different people.  Eg, geochemists may expect full geochemical ability, while petroleum researchers may expect the ability to study reservoir stimulation, while nuclear people would want radionuclide decay.

As per https://github.com/openjournals/joss-reviews/issues/3742

**Comment by WilkAndy** [comment]
Also should probably write what you mean by "stokes", "poisson", etc, for completeness.

**Comment by nasserma** [comment]
We have modified the paper to indicate that the Poisson equation can also be referred to as the Heat equation. Regarding Stokes equations, this is a standard reference to the incompressible Navier-Stokes (INS) equations in the limit of low Reynolds number, so we do not think further clarification is needed and it is not provided in the documentation of other software packages that include it in their examples.

We have changed our naming of the last model to "Multicomponent Mixture INS equations" for clarity. We support inclusion support for first order reactions and are in the progress of documenting support for higher order reactions (nonlinear). Regarding petroleum researchers, they would expect reference to transport in porous media, and we are not very familiar with what you mean about radionuclide decay.
